### PR TITLE
Add AL governor

### LINKED ISSUE #78: Create form for AL Governor
https://contact.governor.alabama.gov/contact.aspx

### PR DESCRIPTION
**Pull request by jason44406**
Fixes #78

### CODE REVIEW

File: states/AL/governor.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          required: true
+          # max length: 200
+        - name: "ctl00$ContentPlaceHolder1$CITY"
+          selector: "#ContentPlaceHolder1_CITY"
+          value: $ADDRESS_CITY
+          required: true
+          # max length: 100
+        - name: "ctl00$ContentPlaceHolder1$STATE"
+          selector: "#ContentPlaceHolder1_STATE"
+          value: $ADDRESS_STATE_FULL
```
Comment (jasonkb) [inline states/AL/governor.yaml]: Should probably be $ADDRESS_STATE_POSTAL_ABBREV?

Comment (jason44406) [reply states/AL/governor.yaml]: The form accepts a full state name.  Is it preferred to always use the abbreviation, even when not limited to 2 characters?

Comment (jasonkb) [reply states/AL/governor.yaml]: Hehe yeah, canonical postal address format in all contexts in the US is with state postal code eh?

Comment (jason44406) [reply states/AL/governor.yaml]: Doesn't matter to me either way.  I guess I was just thinking about how I would personally fill out the form.  I'll change it.

**Pull request merged**